apps/admin-x-activitypub/src/components/Activities.tsx

```diff
@@ -139,14 +140,17 @@ const Activities: React.FC<ActivitiesProps> = ({}) => {
         <>
             <MainNavigation title='Activities' />
             <div className='z-0 flex w-full flex-col items-center'>
-                {activities.length === 0 && (
+                {isLoading ? (
+                    <div className='mt-8 flex flex-col items-center justify-center space-y-4 text-center'>
+                        <LoadingIndicator size='lg' />
+                    </div>
+                ) : activities.length === 0 ? (
```

Comment:
I'm not a fan of nested ternaries, they feel hard to reason about / hard to read. What do you think about doing this in "blocks" instead?

Something like:

```
{
	isLoading && (
	  <div className='mt-8 flex flex-col items-center justify-center space-y-4 text-center'>
	      <LoadingIndicator size='lg' />
	  </div>
	)
}
{
	isLoading === false && activities.length === 0 ? (
		<div className='mt-8'>
            <NoValueLabel icon='bell'>
                When other Fediverse users interact with you, you&apos;ll see it here.
            </NoValueLabel>
        </div>
	)
}
{
	(isLoading === false && activities.length > 0) ? (
		...
	)
}
```

Reply:
@mike182uk yeah this looks easier to understand, I’ll change it 🫡